xml/System.Windows.Forms/ToolStrip.xml

```diff
@@ -181,6 +181,28 @@
         <remarks>To be added.</remarks>
       </Docs>
     </Member>
+    <Member MemberName="AllowClickThrough">
+      <MemberSignature Language="C#" Value="public bool AllowClickThrough { get; set; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance bool AllowClickThrough" />
+      <MemberSignature Language="DocId" Value="P:System.Windows.Forms.ToolStrip.AllowClickThrough" />
+      <MemberSignature Language="VB.NET" Value="Public Property AllowClickThrough As Boolean" />
+      <MemberSignature Language="F#" Value="member this.AllowClickThrough : bool with get, set" Usage="System.Windows.Forms.ToolStrip.AllowClickThrough" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; property bool AllowClickThrough { bool get(); void set(bool value); };" />
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>9.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Boolean</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets or sets a value indicating whether the control to be interacted when window does not have focus.</summary>
+        <value>
+          <see langword="true" /> to allow the control to be interacted when window does not have focus; otherwise, <see langword="false" />. The default value is <see langword="false" />.
+        </value>
+      </Docs>
+    </Member>
```

Comment:
We should not add full API items ourselves. They are added automatically when we feed a new preview or RC version.

I assume this API was added in P7, but right now dotnet-api-docs is showing APIs from P6. Can you please confirm that this API is being added in the new CI update? https://github.com/dotnet/dotnet-api-docs/pull/10257

```suggestion
```

Reply:
Thanks for the information! I was wondering why this one wasn't showing up. Yes, that PR adds this API among some other System.Drawing ones. I will wait until that PR goes in then and pull in the changes.